playground/src/initial.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button, Frog } from 'frog'
+import { Heading, VStack, vars } from './ui.js'
+
+export const app = new Frog({
+  ui: { vars },
+})
+  .frame('/', (c) => {
+    return c.res({
+      image: '@/refreshing-image',
```

Comment:
@jxom, I was forced to add some kind of prefix to differentiate this from static assets that might be added via passing `assetsUrl`.
https://github.com/wevm/frog/blob/a98f567e878611609911dcca9abb28f000d1ea4a/src/frog-base.tsx#L589